src/include/rados/objclass.h

```diff
@@ -0,0 +1,183 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef CEPH_OBJCLASS_OBJCLASS_PUBLIC_H
+#define CEPH_OBJCLASS_OBJCLASS_PUBLIC_H
+
+#ifdef __cplusplus
+
+#include <map>
+#include <set>
```

Comment:
Yes, they are not required now. I had included them for cls_cxx_getxattrs() and cls_cxx_map_get_keys(), which are no more a part of the api. I will remove them. Thanks for pointing that.